e2e/cypress/integration/system_console/ui_and_api/customization_spec.js

```diff
@@ -396,6 +396,31 @@ describe('Customization', () => {
             expect(config.TeamSettings.EnableCustomBrand).to.equal(false);
         });
     });
+
+    it('T1024 - Can change name and desc with Custom Branding set to false', () => {
```

Comment:
Wondering if this should go in another file, like `e2e/cypress/integration/system_console/site_configuration/customization_spec.js`, to respect TM4J folder structure

/cc @saturninoabril

Reply:
@mgdelacroix Yes, please. Thanks for asking.

Also, please include full test ID - `MM-T1024`
```suggestion
    it('MM-T1024 - Can change name and desc with Custom Branding set to false', () => {
```